test/WebJobs.Script.Tests.Shared/TestHelpers.cs

```diff
@@ -42,6 +43,22 @@ public static string FunctionsTestDirectory
             }
         }
 
+        public static Task WaitOneAsync(this WaitHandle waitHandle)
+        {
+            if (waitHandle == null)
+            {
+                throw new ArgumentNullException("waitHandle");
+            }
+
+            var tcs = new TaskCompletionSource<bool>();
+            var rwh = ThreadPool.RegisterWaitForSingleObject(waitHandle,
+                delegate { tcs.TrySetResult(true); }, null, -1, true);
+            var t = tcs.Task;
+            t.ContinueWith((antecedent) => rwh.Unregister(null));
+
+            return t;
```

Comment:
Did you mean to return the task returned by the `ContinueWith` call above? Any other continuations should come after that one.

Reply:
I'm using code from official documentation here: https://docs.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/interop-with-other-asynchronous-patterns-and-types?redirectedfrom=MSDN#from-wait-handles-to-tap. Agree with you that that code should return the continuation. Making that change.